packages/mobile/src/localCurrency/saga.test.ts

```diff
@@ -0,0 +1,80 @@
+import { expectSaga } from 'redux-saga-test-plan'
+import * as matchers from 'redux-saga-test-plan/matchers'
+import { throwError } from 'redux-saga-test-plan/providers'
+import { call } from 'redux-saga/effects'
+import { waitForRehydrate } from 'src/app/saga'
+import {
+  fetchCurrentRateFailure,
+  fetchCurrentRateStart,
+  fetchCurrentRateSuccess,
+} from 'src/localCurrency/actions'
+import { fetchExchangeRate, updateLocalCurrencyRate } from 'src/localCurrency/saga'
+import rootReducer from 'src/redux/reducers'
+import { createMockStore } from 'test/utils'
+
+const now = Date.now()
+Date.now = jest.fn(() => now)
```

Comment:
We should probably restore this after your tests are done

Reply:
Yes I wondered about this too. I borrowed this from another [test we already had](https://github.com/celo-org/celo-monorepo/blob/b7ad7840e6d37594a89dad2177b7eab6786150cc/packages/mobile/src/stableToken/saga.test.ts#L14).
From what I understand, tests within a file are isolated from other test files we have, so this is fine. What do you think?

Reply:
Ok np then